stored_requests/events/api/api.go

```diff
@@ -0,0 +1,65 @@
+package api
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/stored_requests/events"
+)
+
+type eventsAPI struct {
+	invalidations chan []string
+	updates       chan map[string]json.RawMessage
+}
+
+// NewEventsAPI creates an EventProducer that generates cache events from HTTP requests.
+// The returned httprouter.Handle must be registered on both POST (update) and DELETE (invalidate)
+// methods and provided an `:id` param via the URL, e.g.:
```

Comment:
Might want to add a comment that this should not be exposed as an endpoint without some kind of authentication as that would allow anyone to `POST` malicious configs to the cache.